tests/python/unit_tests/test_optimizers/test_pybernyop.py

```diff
@@ -18,7 +18,15 @@
 import pluginplay as pp
 import chemist
 import unittest
-from simde import TotalEnergy
+from simde import TotalEnergyNuclearOptimization
+
+
+def threed_distance(coords):
```

Comment:
This function name is very misleading and the implementation is not general. I'd recommend:
 
```suggestion
def diatomic_bond_distance(coords):
```